src/background.js

```diff
@@ -0,0 +1,34 @@
+BROWSER_SDK.storage.sync.get("domains", function(items) {
```

Comment:
`BROWSER_SDK` is injected at build time with either `browser` for Firefox or `chrome` for Chrome.